_documentation/user/flags/forward-only.md

```diff
@@ -5,4 +5,19 @@ type: Bad
 summary: Can't drive in reverse.
 ---
 
-...
+The Forward Only flag prevents backwards movement, as if a tank has Oscillation-Overthruster and is within a building.
+
+## Tactics
+
+Forward Only makes it harder to dodge and generally reduces efficiency in movement.
+
+Countering the effects of Forward Only requires good map knowledge of areas forward movement is restricted or where backwards movement is required for efficient movement.
+This allows one to be able to avoid being in dangerous or risky positions, which reduces the danger and risk, while generally making it more difficult for the enemy as well.
+The main danger to Forward Only is being in areas where movement is inefficient or time-consuming, which generally makes one an easy target.
+Another major risk is tanks which are approaching from the rear, as it is normally almost impossible to out-maneuver them at close range.
+By avoiding the risks and dangers of Forward Only, one will generally be able to well with it.
```

Comment:
This whole paragraph is very vague. Be more descriptive about the "dangerous or risky positions." When do tanks typically need to go backward? What happens when a tank with FO gets stuck in a corner?

Reply:
Will expand with detailed info.